docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  app:
+    container_name: review
+    restart: on-failure
+    build: .
+    ports:
+      - '80:3003'
```

Comment:
How come we need port 80 here?

Reply:
oh my 3003 was taken at that time from something so i used 80 for that time i will change back to 3003